deploy/platform/kubernetes/Makefile.in

```diff
@@ -21,4 +21,4 @@
 NAMESPACE ?= default
 AGENTLESS ?= false
 
-FEATURE_FLAGS ?= java-agent-injector,cluster,kubernetes-monitor,so11y,vm,als,event,istiod-monitor
\ No newline at end of file
+FEATURE_FLAGS ?= java-agent-injector,cluster,kubernetes-monitor,so11y,vm,als,event,istiod-monitor,satellite
```

Comment:
Please disable satellite by default

Reply:
Do we have a proper way to set agent backend address when satellite is or is not disable